doc/how_to/custom_components/index.md

```diff
@@ -12,20 +16,109 @@ These How-to pages provide solutions for common tasks related to extending Panel
 How to build custom components that are combinations of existing components.
 :::
 
-:::{grid-item-card} {octicon}`plus-circle;2.5em;sd-mr-1 sd-animate-grow50` Build Components from Scratch
-:link: custom_reactiveHTML
+::::
+
+## `ReactiveHTML` Components
```

Comment:
This section is currently broken

![image](https://github.com/holoviz/panel/assets/42288570/39299226-6411-4f72-ae3d-7b28a1e8dbf0)